app/routes/__index/dns-records/$dnsRecordId.tsx

```diff
@@ -61,7 +64,7 @@ export const action = async ({ request }: ActionArgs) => {
     id: Number(data.id),
     username: user.username,
     type: data.type,
-    subdomain: data.subdomain,
+    subdomain: filter.clean(data.subdomain),
```

Comment:
Do you think it would be better to show an error on the UI if these words exist in the subdomain instead of saving the record with them filtered out?

Reply:
Thanks for the comment! I changed it to decline inappropriate subdomain with proper message. Please review